cmd/peer-rpc-client-target.go

```diff
@@ -31,11 +31,16 @@ func (target *PeerRPCClientTarget) ID() event.TargetID {
 	return target.id
 }
 
-// Send - sends event to remote peer by making RPC call.
-func (target *PeerRPCClientTarget) Send(eventData event.Event) error {
+// Save - sends event to remote peer by making RPC call.
+func (target *PeerRPCClientTarget) Save(eventData event.Event) error {
 	return target.rpcClient.SendEvent(target.bucketName, target.id, target.remoteTargetID, eventData)
 }
 
+// Send - sends event to Client target.
+func (target *PeerRPCClientTarget) Send(eventKey string) (bool, error) {
```

Comment:
You could call `Send()` in `Save()`.

Reply:
The parameter type doesn't match here. Send accepts a string but Save requires event.Event. Anyway, this will prolong only for a shore time as i will implement this in other targets shortly in fore-coming PRs.

Reply:
I am saying like below
```go
// Save - sends event immediately instead of saving it.
func (target *PeerRPCClientTarget) Save(eventData event.Event) error {
	return client.send(eventData)
}

func (target *PeerRPCClientTarget) send(eventData event.Event) error {
	return target.rpcClient.SendEvent(target.bucketName, target.id, target.remoteTargetID, eventData)
}

// Send - interface compatible method does no-op.
func (target *PeerRPCClientTarget) Send(eventKey string) error {
	return nil
}
```

This pattern can be used targets those does immediate send i.e. not using event store.

Reply:
understood :+1: